pkg/client/jiva/model.go

```diff
@@ -50,7 +49,7 @@ type InfoReplica struct {
 	Chain           []string            `json:"chain"`
 	Disks           map[string]DiskInfo `json:"disks"`
 	RemainSnapshots int                 `json:"remainsnapshots"`
-	RevisionCounter string              `json:"revisioncounter"`
+	RevisionCounter int64               `json:"revisioncounter"`
```

Comment:
why is this data type change required?

Reply:
It should be `string`, i think while refactoring/replace(duplicate struct), had deleted the struct which was getting used before.

Reply:
done !